pkg/testutils/kyma.go

```diff
@@ -117,10 +198,11 @@ func IsKymaInState(ctx context.Context, name, namespace string, clnt client.Clie
 		string(state))
 }
 
-func ExpectKymaManagerField(
-	ctx context.Context, controlPlaneClient client.Client, kymaName string, managerName string,
+func ContainsExpectKymaManagerField(
```

Comment:
Consider renaming it to `ContainsKymaManagerField` or `StatusContainsManagerField`

Reply:
changed to ContainsKymaManagerField